src/Umbraco.Core/Media/EmbedProviders/Twitter.cs

```diff
@@ -14,7 +14,7 @@ public Twitter(IJsonSerializer jsonSerializer)
 
     public override string ApiEndpoint => "http://publish.twitter.com/oembed";
 
-    public override string[] UrlSchemeRegex => new[] { @"twitter.com/.*/status/.*" };
+    public override string[] UrlSchemeRegex => new[] { @"(twitter|x).com/.*/status/.*" };
```

Comment:
This is a problem with all embedded providers, but because the new URL is so badly chosen, it'll likely cause a lot of false positives (all domains ending with x.com + /status will match).

Reply:
Do you have any suggestions on how to amend the regex in this case?

Reply:
@elit0451 I think this Regex could capture the essence, but might not be bulletproof:

```
(?:https?:\/\/)(?:www\.)?(twitter|x)\.com\/.*\/status\/.*
```

You can see the tests here, where it matches everything except "testx.com":

<img width="586" alt="image" src="https://github.com/umbraco/Umbraco-CMS/assets/752371/1b099ecc-389e-445c-804c-c7012a57cefb">

Reply:
Cool, I'll update. Thank you 🙌